astropy/samp/tests/web_profile_test_helpers.py

```diff
@@ -195,12 +195,12 @@ def register(self):
 
             if result["samp.self-id"] == "":
                 raise SAMPClientError(
-                    "Registation failed - samp.self-id was not set by the hub."
+                    "Registration failed - samp.self-id was not set by the hub."
```

Comment:
@mhvk @pllim, thoughts on the message change? IMO it's fine to correct the spelling, esp in SAMP.

Reply:
I don't see how fixing typo is bad?

Reply:
Agreed. but there's been a lively debate about changing error messages. Don't want to step on toes 😆.